_data/cards.yml

```diff
@@ -0,0 +1,37 @@
+develop:
+  - link: "/cockroachcloud/quickstart.html"
+    title: "Create a new cluster"
+    text: "Create and connect to a new CockroachDB cluster."
+  - link: "/stable/install-client-drivers.html"
+    title: "Install drivers"
+    text: "Install client drivers for your application."
+  - link: "/stable/connect-to-the-database.html"
+    title: "Connect to a cluster"
+    text: "Get details on how to connect to a CockroachDB cluster using your favorite language and framework."
+  - link: "/stable/example-apps.html"
+    title: "Example applications"
+    text: "Browse our example applications written in your favorite language and framework."
+  - link: "/stable/tutorials-overview.html"
+    title: "Tutorials"
+    text: "Browse our tutorials."
+  - link: "/stable/developer-basics.html"
+    title: "Developer basics"
+    text: "A quick overview of key concepts developers need to know about CockroachDB."
+  - link: "cockroachcloud/learn-cockroachdb-sql.html"
+    title: "Learn CockroachDB SQL"
+    text: "Submit SQL queries to retrieve data from your cluster."
+  - link: "/stable/schema-design-overview.html"
+    title: "Design a schema"
+    text: "Details and best practices about designing schemas for your data."
+  - link: "/stable/query-data.html"
+    title: "Read data"
+    text: "How to query your cluster to retrieve data."
+  - link: "/stable/insert-data.html"
```

Comment:
This is kind of a longstanding issue in our docs, but I was a little confused when I clicked on "Write Data" and the page I landed on was called "Insert Data".  Then I was not confused b/c I know this is an issue we have not yet decided to resolve, but even knowing about it I always feel a "glitch in the matrix" for a second when this happens.  

I worry that the glitch is even more pronounced for our users who are not as familiar with how our docs are structured internally.  It kinda breaks the mental model of "where I was and what I clicked vs. where I am now" IMO

this PR is not the place to solve this longstanding issue but I'm bringing it up b/c it is still a bit jarring when I encounter it and this is yet another place we're adding that tickles the issue

Reply:
personally I'd be fine with this PR editing "Insert Data" to call it "Write Data" and updating other links throughout the docs for consistency, but understandable if you disagree

Reply:
same comment also applies to "read data" vs "query data" and a few others that I'm sure you already know about

Reply:
I agree with you, but I think that change is going to overwhelm this PR. Let's discuss this separately.